nl/nl_linux.go

```diff
@@ -615,7 +615,10 @@ func (s *NetlinkSocket) Receive() ([]syscall.NetlinkMessage, error) {
 	if fd < 0 {
 		return nil, fmt.Errorf("Receive called on a closed socket")
 	}
-	rb := make([]byte, unix.Getpagesize())
+	// Set a big temporary buffer because returned message from kernel can
+	// be quite verbose if kernel is asked e.g. for statistics,
+	// tc rules or filters, or other more memory requiring data.
+	rb := make([]byte, 65536)
```

Comment:
@jellonek, I will test later today.

Reply:
Please define a constant for this.

Reply:
@jellonek , sorry for the late response. Something came up.

I tested now with 100 VFs invalid request error is not present with this change.

Please define a const and also add comment there.

Reply:
Thx for the test.
Const added and commented.